workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css

```diff
@@ -692,6 +692,19 @@ pre[class*='language-'] {
 	}
 }
 
+.search-banner {
```

Comment:
I am not sure we want/need this new class. we are defining .learn-banner in this file, and should not be overwriting those values. If we need to update the values we should do so in learn-banner which applies everywhere. Unless we really want the search banner to be different then the banner on home and other page (which i would be surprised if thats the case.

Reply:
hey ryan, the search banner is different from the home banner, the reason why I added this class. The search banner component has the learn-banner class (base style) + search-banner class (size adjustments).